lms/djangoapps/certificates/api.py

```diff
@@ -0,0 +1,118 @@
+"""
+Certificates API views
```

Comment:
Usually we try to put view-level functions (which accept a `request`) in `views.py`, and we put application logic (which does not accept a request) in `api.py`.  Could you change this docstring so it doesn't have the word "views" in it?